File: monitor/api-error.ts
import axios from 'axios';

export function sendMonitorError(monitor: string, content: string): void {
  const body = { phones: ["13917926210"], content: `error monitor ${monitor}: ${content}` }
  axios.post(`http://notice.cmsfs.org:3000/v1/notice/phone`, body).catch(e => console.error(e))
  return
}

export function sendSystemError(content: string) {
  axios.post(`http://notice.cmsfs.org:3000/v1/notice/phone`, content).catch(e => console.error(e)).catch(e => console.error(e))
  return
}

File: monitor-oracle/api_error.py
import json
import http.client

def sendError(monitor, content):
    headers = {"Content-type": "application/json; charset=utf-8"}
    conn = http.client.HTTPConnection("notice.cmsfs.org", 3000)
    url = "/v1/notice/phone"
    content = json.dumps({"phones": ["13917926210"],"content":"montior: %s, error: %s" % (monitor,content)})
    conn.request("POST", url, content, headers)
    response = conn.getresponse()
    print(response.status, response.reason)
    conn.close()